learn/glossary.md

```diff
@@ -14,6 +14,15 @@ If you encounter a term you wish were defined here, please feel free to [file an
 
 The entries on this page can be linked to via anchor links (e.g. `https://json-schema.org/learn/glossary.html#vocabulary`) when sharing a definition with others.
 
+### dialect
+
+A collection of [vocabularies](#vocabulary), along with an indication of whether supporting each vocabulary is required to process schemas written in the dialect.
```

Comment:
I think the part about vocabularies potentially being optional is a Vocabulary System concept. I'd remove it.

```suggestion
A collection of [vocabularies](#vocabulary) that identify the set of keywords an implementation needs to understand to process a schema.
```

Reply:
What definition of "Vocabulary System" are you using that results in you excluding optionality?  Have you built consensus around it?  The optionality of vocabularies is hardly an implementation detail.  And again, it is part of what is real _now_.

Reply:
What about taking @jdesrosiers's wording but using "should" instead of "needs to"?

Reply:
I think the original wording was clear as plain language.  Getting into "should" being a way to "suggest" something feels more spec-ese to me.